src/pages/blogs/[id].tsx

```diff
@@ -21,15 +23,40 @@ export async function getStaticPaths() {
 }
 
 export default function Post({ postData }) {
+    const breakPoint = useMediaQuery('(min-width:600px)');
+
     return (
-        <div>
-            <h1>{postData.title}</h1>
+        <Box padding="0 7.5% 100px" bgcolor={bgColor} color="white">
```

Comment:
The other sides are fine, but I would suggest adding an appropriate padding at the top too. You can decide how much padding from trial and error.